source/core/parameters_cfd_dem.cc

```diff
@@ -19,6 +19,11 @@ namespace Parameters
                       "false",
                       Patterns::Bool(),
                       "Define particles using a DEM simulation results file.");
+    prm.declare_entry(
```

Comment:
Could we move this file instead to the fem-dem library? It does not need to lie in core.

Reply:
done